website/docs/guides/oidc.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Common OIDC provider configurations
+---
+
+This page provides guides for configuring Weave GitOps with the most common OIDC providers.
+
+## Google
+
+Google's identity provider does not support the groups scope which Weave GitOps requests by default. That's why in
+this example the `customScopes` field is set to only request the `openid` and `email` scopes.
+
+1. Obtain the client ID and secret by following the [official guide](https://developers.google.com/identity/openid-connect/openid-connect)
+   from Google.
+1. Configure Weave GitOps:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+       name: oidc-auth
+       namespace: WEAVE_GITOPS_NAMESPACE
+   stringData:
+       clientID: CLIENT_ID_FROM_STEP_1
+       clientSecret: CLIENT_SECRET_FROM_STEP_1
+       issuerURL: https://accounts.google.com
+       redirectURL: BASE_WEAVE_GITOPS_URL/oauth2/callback
+       customScopes: openid,email
+   ```
+
+## Azure AD
+
+1. Obtain the client ID and secret by following the [official guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+   from Microsoft.
+1. [optional] Configure group claims by following this [official guide](https://learn.microsoft.com/en-us/security/zero-trust/develop/configure-tokens-group-claims-app-roles).
+1. Configure Weave GitOps:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+       name: oidc-auth
+       namespace: WEAVE_GITOPS_NAMESPACE
+   stringData:
+       clientID: CLIENT_ID_FROM_STEP_1
+       clientSecret: CLIENT_SECRET_FROM_STEP_1
+       issuerURL: https://login.microsoftonline.com/TENANT_ID/v2.0
+       redirectURL: BASE_WEAVE_GITOPS_URL/oauth2/callback
+       customScopes: openid
+       claimUsername: sub
+   ```
+
+## Keycloak
+
+Keycloak is highly customizable so the steps to obtain client ID and secret will vary depending on your setup. The
+general steps are very similar and the following steps point to the appropriate pages in the official Keycloak
+documentation:
+
+1. Log in to the Keycloak admin console and [create a realm](https://www.keycloak.org/docs/latest/server_admin/#configuring-realms).
+1. [Create a client application](https://www.keycloak.org/docs/latest/authorization_services/index.html#_resource_server_create_client)
+   and choose "OpenID Connect" as the client type.
+1. Make sure to set the "Client Authenticator" on the "Credentials" tab to "Client Id and Secret" and generate a secret.
```

Comment:
Do we need to say anything about scopes for the client?

Do we need to say anything about groups for this? (realm roles?!)

Reply:
I basically stopped iterating on this part of the page as soon as #4113 arrived because that PR provides all the nitty-gritty details of using WeGO with Keycloak. So I suppose this part will be overridden soon, anyway.

Reply:
Gotcha, maybe we can remove this?

Reply:
And link to the other doc?

Reply:
Yes, I will as soon as the other doc is merged.